apps/admin/backend/src/app.write_ins.test.ts

```diff
@@ -337,6 +345,67 @@ test('getWriteInImageView', async () => {
   `);
 });
 
+test('getWriteInImageView on bmd', async () => {
+  const { auth, apiClient } = buildTestEnvironment();
+  const { electionDefinition, castVoteRecordExport } =
+    electionTwoPartyPrimaryFixtures;
+  await configureMachine(apiClient, auth, electionDefinition);
+
+  const reportDirectoryPath = castVoteRecordExport.asDirectoryPath();
+  (
+    await apiClient.addCastVoteRecordFile({
+      path: reportDirectoryPath,
+    })
+  ).unsafeUnwrap();
+
+  // look at a contest that can have multiple write-ins per ballot
+  const contestId = 'zoo-council-mammal';
+  const writeInIds = await apiClient.getWriteInAdjudicationQueue({
+    contestId,
+  });
+  expect(writeInIds).toHaveLength(24);
+  const [writeInIdA, writeInIdB] = writeInIds.slice(0, 2);
+  assert(writeInIdA !== undefined && writeInIdB !== undefined);
+
+  // check image of first write-in
+  const writeInImageViewA = await apiClient.getWriteInImageView({
+    writeInId: writeInIdA,
+  });
+  assert(writeInImageViewA);
+
+  const { imageUrl: actualImageUrl, machineMarkedText: machineMarkedTextA } =
+    writeInImageViewA as BmdWriteInImageView;
+
+  expect(machineMarkedTextA).toEqual('Mock Write-In');
+
+  const expectedImage = await loadImage(
+    join(
+      reportDirectoryPath,
+      '02daef18-5915-423a-baba-4130920662d9',
+      '02daef18-5915-423a-baba-4130920662d9-front.jpg'
```

Comment:
Will this identifier change when regenerating CVR fixtures? If so, might be worth thinking about how to make the test resilient to that change

Reply:
oh I see now the test I was copying this from was using a manual export... I don't think this actually needs to be tested as its the same code path that is used for hmpb for this piece so will just remove